src/site/layouts/va_form.drupal.liquid

```diff
@@ -24,38 +24,46 @@
           <h2 class="vads-u-margin-top--0 vads-u-margin-bottom--4">{{ fieldVaFormName }}</h2>
         </hgroup>
 
-        {% if fieldBenefitCategories.length > 0 or fieldVaFormRevisionDate %}
-          <dl>
-            {% if fieldBenefitCategories.length > 0 %}
-              <div>
-                <dt class="vads-u-font-weight--bold vads-u-display--inline">
-                  Related to:
-                </dd>
-                <dd class="vads-u-display--inline">
+        <dl>
+          <div class="vads-u-margin-y--1">
+            <dt class="vads-u-font-weight--bold vads-u-display--inline">
+              Related to:
+            </dd>
+            <dd class="vads-u-display--inline">
+              {% if fieldBenefitCategories.length > 0 %}
+
+                {% if fieldBenefitCategories.first == 'Non-VA' %}
+                  A non-VA form. For other government agency forms, go to the <a href="https://www.gsa.gov/reference/forms">GSA forms library</a>.
+                {% elsif fieldBenefitCategories.first == 'Employment/job' %}
+                  Employment or jobs at VA
+                {% else %}
                   {{ fieldBenefitCategories | map: "entity" | map: "entityLabel" | join: '; ' }}
-                </dd>
-              </div>
-            {% endif %}
-
-            {% if fieldVaFormRevisionDate %}
-              <div>
-                <dt class="vads-u-font-weight--bold vads-u-display--inline">
-                  Form last updated:
-                </dt>
-                <dd class="vads-u-display--inline">
-                  {{ fieldVaFormRevisionDate.value | humanizeDate }}
-                </dd>
-              </div>
-            {% endif %}
-          </dl>
-        {% endif %}
+                {% endif %}
+
+              {% else %}
+                {{ fieldVaFormAdministration.entity.entityLabel }}
+              {% endif %}
+            </dd>
+          </div>
+
+          {% if fieldVaFormRevisionDate %}
+            <div>
+              <dt class="vads-u-font-weight--bold vads-u-display--inline">
+                Form last updated:
+              </dt>
+              <dd class="vads-u-display--inline">
+                {{ fieldVaFormRevisionDate.value | humanizeDate }}
+              </dd>
+            </div>
+          {% endif %}
+        </dl>
 
         {% if fieldVaFormUsage %}
           <h3>When to use this form</h3>
           {{ fieldVaFormUsage.processed }}
         {% endif %}
 
-        <a href="{{ fieldVaFormUrl.uri }}">Download VA Form {{ fieldVaFormNumber }} (PDF)</a>
+        <a href="{{ fieldVaFormUrl.uri }}" download>Download VA Form {{ fieldVaFormNumber }} (PDF)</a>
```

Comment:
> All Download links on the form detail page - make them initiate download, same as with download links on forms search

We're gonna run into the same problem as on Find Forms though where IE doesn't support the `download` attribute. We'll have to circle back around.

Reply:
Fixed by chunking out the Find Forms download logic into a polyfill